edge-detect/yolo/base_mqtt_pub_sub.py

```diff
@@ -0,0 +1,319 @@
+"""This file include the parent class for all future edge technology work by IQT Labs that
```

Comment:
@meadej Can this be pulled from the source rather than copied?  Same comment as before, I would like this to come from source control

Reply:
Absolutely can. Implemented in https://github.com/IQTLabs/edgetech-coral-detect/pull/1